Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM python:3.8.6-buster as base
+RUN pip install "poetry==1.1.4"
+WORKDIR /todo-app
+EXPOSE 5000
+
+FROM base as production
+COPY . /todo-app

```

Comment:
To optimise rebuilding images, you could COPY the poetry/toml files first (in the base stage even), and then move `COPY . /todo-app` after the `poetry install` command, for both the production and test stages.